sakura_core/util/string_ex.h

```diff
@@ -239,6 +243,11 @@ char*	wcstombs_new(const wchar_t* pSrc,int nSrcLen); //戻り値はnew[]で確
 void	wcstombs_vector(const wchar_t* pSrc, std::vector<char>* ret); //戻り値はvectorとして返す。
 void	wcstombs_vector(const wchar_t* pSrc, int nSrcLen, std::vector<char>* ret); //戻り値はvectorとして返す。
 
+std::string wcstombs(std::string& strOut, std::wstring_view strInput);
+std::wstring u8stowcs(std::wstring& strOut, std::string_view strInput);
```

Comment:
`wcstombs`は`wcstou8s`の間違いではないかと。
```suggestion
std::wstring u8stowcs(std::wstring& strOut, std::string_view strInput);
std::string wcstou8s(std::string& strOut, std::wstring_view strInput);
```

Reply:
ちょっと確認します。

追加してるのは `u8s` と `wcs` の相互変換で、
今回は `mbs` に関連する修正を入れてないつもりです（次のPRで入れたいです・・・

Reply:
誤記と確認できたので修正しました。

> wcstombsはwcstou8sの間違いではないかと。

はい、そうです。cppと書き順をあわせているので書き順も入れ替えませ👇。